code/modules/mob/living/simple_animal/hostile/statue.dm

```diff
@@ -2,7 +2,7 @@
 
 /mob/living/simple_animal/hostile/statue
 	name = "statue" // matches the name of the statue with the flesh-to-stone spell
-	desc = "An incredibly lifelike marble carving. Its eyes seem to follow you.." // same as an ordinary statue with the added "eye following you" description
+	desc = "An incredibly lifelike marble carving. It's eyes seem to follow you..." // same as an ordinary statue with the added "eye following you" description
```

Comment:
"it's" means 'it is' or 'it has'
"its" was fine in this case

Reply:
It's eyes as in the statue's eyes

Reply:
possesive form is its
it's means it has or it is

Reply:
Both are valid for possessive form, the one without the apostrophe is better IMO for explicit purpose of distinguishing it from contraction of "it is" (contractions MUST have the apostrophe).

Reply:
we've been owned! he's right!

Reply:
god fucking dammit